mikelpardo/mobile-sales
# Get mobile list from endpoint and add tests to the home view

**Description**

Get all mobiles from given endpoint and add core and home view tests

**How to test**

Go to: http://localhost:3000/
Check in the console that it gets all mobile list from the endpoint and display correctly in the view
